JS/Lesson34/goksel/recipe-app/src/modules/recipes/RecipeDetail/index.tsx

```diff
@@ -0,0 +1,76 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import CardMedia from '@mui/material/CardMedia';
+import Typography from '@mui/material/Typography';
+import { CardActionArea, Stack } from '@mui/material';
+import Alert from '@mui/material/Alert';
+import { Recipe } from "../models";
+
+export const RecipeDetail = () => {
+    const { id } = useParams();
+
+    const [recipeDetail, setRecipeDetail] = useState<Recipe|null>(null);
+
+
+    useEffect(() => {
```

Comment:
It would be good to search for the recipe in the redux store first, if it's not found, then make the API call.

Reply:
Fixed.